### ISSUE TITLE
Partially incorrect instructions in README.md

### ISSUE THREAD

**Issue by olvovitch** [lightstep/lightstep-tracer-java#166]
https://github.com/lightstep/lightstep-tracer-java/blob/master/README.md indicates that the following dependencies should be included:
```
<dependency>
    <groupId>com.lightstep.tracer</groupId>
    <artifactId>lightstep-tracer-jre</artifactId>
    <version> VERSION </version>
</dependency>

<dependency>
    <groupId>com.lightstep.tracer</groupId>
    <artifactId>tracer-grpc</artifactId>
    <version> VERSION </version>
</dependency>
```

This is not, actually, correct. `VERSION` is the version of `lightstep-tracer-jre` and currently is set to 0.14.8. There's no such thing as `tracer-grpc` with that version. Instead, the version of https://github.com/lightstep/lightstep-tracer-java-common should be used (currently `0.15.10`)

Basically, either the versions of `lightstep-tracer-jre` and `lightstep-tracer-java-common` should be kept in sync, or instructions should be updated.

**Comment by carlosalberto** [comment]
This is fixed now. Feel free to re-open if you think we need further details ;)
